jdaviz/configs/default/plugins/export_plot/export_plot.py

```diff
@@ -20,16 +33,41 @@ class ExportViewer(PluginTemplateMixin, ViewerSelectMixin):
     * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
     * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
     * :meth:`save_figure`
+    * :meth:`save_movie` (Cubeviz only)
     """
     template_file = __file__, "export_plot.vue"
 
+    # For Cubeviz movie.
+    i_start = Any(0).tag(sync=True)
+    i_end = Any(0).tag(sync=True)
+    movie_fps = Any(5.0).tag(sync=True)
+    movie_filename = Any("mymovie.mp4").tag(sync=True)
+    movie_msg = Unicode("").tag(sync=True)
+    movie_recording = Bool(False).tag(sync=True)
+    movie_kill = Bool(False).tag(sync=True)
+
     @property
     def user_api(self):
+        if self.config == "cubeviz":
+            return PluginUserApi(self, expose=('viewer', 'save_figure', 'save_movie'))

```

Comment:
```suggestion
            return PluginUserApi(self, expose=('viewer', 'save_figure', 'save_movie',
                                                                         'i_start', 'i_end', 'movie_fps', 'movie_filename'))

```

(API docstring would also need to be updated to include these)

Reply:
Why? What if someone uses the API to set an invalid value of `i_start` and `i_end` combo? Just more things we have to guard against if they are public. Is `save_movie` not enough? It takes `i_start` and `i_end` as arguments.

Reply:
Because we are trying to strive for reproducible plugin workflows through the API and all other plugins do so by exposing all input widgets.  I get that it is possible here to do the same thing by passing the kwargs, but now the user can't even pull the values set in the widgets.

Reply:
Is there a way to have them pull it as read-only? Or you want them to be settable via API on purpose?

Reply:
The rest of the plugin APIs allow setting all public traitlets and there is no current way to expose them as read-only, although I suppose it would be possible to create an infrastructure for that.  Since the method checks the values anyways, what would be the harm in letting the user set them via traitlets?

Reply:
I added this as part of a larger change but I'll add you as co-author. I think it is ok to let user change it, because I also added validation to Vue.js so hopefully enough warning that way.